openshift/Makefile

```
@@ -7,6 +7,16 @@ include $(addprefix $(DIR)/vendor/github.com/openshift/build-machinery-go/make/,
 
 include $(DIR)/.bingo/Variables.mk
 
+# Prefer the upstream source commit that the Dockerfile passes in via
+# `ENV GIT_COMMIT=<sha>`. If that variable is not already defined fall back to
+# the commit recorded by the OpenShift image build pipeline.
+ifeq ($(origin GIT_COMMIT), undefined)
+GIT_COMMIT := $(OPENSHIFT_BUILD_COMMIT)   # populated by OpenShift build machinery
+endif
+export GIT_COMMIT
+VERSION_PATH            := github.com/operator-framework/operator-controller/internal/shared/version
+export GO_BUILD_LDFLAGS := -s -w -X '$(VERSION_PATH).gitCommit=$(GIT_COMMIT)'  #
```

```suggestion
export GO_BUILD_LDFLAGS := -s -w -X '$(VERSION_PATH).gitCommit=$(GIT_COMMIT)' 
```

NIT: I think # is here by mistake right?

---

yep, mistake, thanks for the catch. Changing in next commit.